resources/README.md

```diff
@@ -0,0 +1,45 @@
+### [Official Website](https://webassembly.org)
+
+## Documentation
+
+#### Specifications
+ - [Core on webpage](https://webassembly.github.io/spec/core)
+ - [Core on PDF](https://webassembly.github.io/spec/core/_download/WebAssembly.pdf)
```

Comment:
I think that there is no need in having PDF link here, but JS API, Web API, and WASI specs would be helpful. https://webassembly.org/specs/ could be enough here